### PR TITLE
Email styling

### PR DESCRIPTION
**Pull request by samhstn**
Merge after #173 
See #59 

Adding media query so the button is over the edge of the card

### CODE REVIEW

**Comment by nelsonic** [comment]
dwylbot: PR _cannot_ be assigned until CI is passing. 👍

**Review by Danwhy — APPROVED** [review]

**Comment by iteles** [comment]
@SimonLab Reassigning this to @nelsonic so we can get it in (if there are no review comments) before our 10am demo.

File: web/static/css/app.css [hunk 1]
```diff
@@ -278,3 +278,9 @@ code {
 .b--lm-green {
   border-color: #81c03d;
 }
+
+@media only screen and (min-width: 950px) {
+  #alphasection button {
+    bottom: -30px;
```
Comment (nelsonic) [inline web/static/css/app.css]: these kinds of "_negative_" numbers should always be accompanied by an in-line comment  clarifying _why_ we are applying the style.
While I'm _OK_ with merging this for the Demo, I think we need a _process_ for this.
cc: @iteles

**Review by nelsonic — APPROVED** [review]
@shouston3 thanks for applying this style.
Would prefer to have a comment next to the style explaining _why_ we need `-30px` to be applied.
Merging for demo.
Thanks.

**Pull request merged**